public/index.html

```diff
@@ -5,9 +5,16 @@
         <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
-        <meta name="description" content="Web site created using create-react-app" />
+        <meta name="description" content="Her kan du klage på vedtak." />
+
+        <meta property="og:title" content="Klagesystem - www.nav.no" />
+        <meta property="og:image" content="http://familie.nav.no/dist/assets/foreldrepenger.png" />
+        <meta property="og:url" content="https://klage-dittnav.dev-nav.no" />
+        <meta name="twitter:card" content="http://familie.nav.no/dist/assets/foreldrepenger.png" />

```

Comment:
Hei. Dette er ikke helt mitt domene, så tar sjansen til å stille noen spørsmål. Kanskje jeg lær meg noe :)

Disse feltene er midlertidige da, siden det er hardkodet til foreldrepenger og dev-nav.no, regner jeg med. Disse "og"-tingene er nytt for meg, men lurer på om noen av appens sider kommer å være foran passord? D.v.s. er det noen av denne appens sider som man skal kunne lenke til? Eller er det andre fordeler med Open Graph?